terraform/prod/terragrunt.hcl

```diff
@@ -40,5 +40,7 @@ generate "prod_tfvars" {
   forest_client_api_base_url = "https://nr-forest-client-api-prod.api.gov.bc.ca/"
   use_override_proxy_endpoints = false
   idim_proxy_api_base_url = "https://nr-fam-idim-lookup-proxy-prod-backend.apps.silver.devops.gov.bc.ca"
```

Comment:
Admin management apis are still using this variable, we could remove it once we figure out how to update the router guard to verify users with IDIM url based on app env